apiserver/facades/agent/provisioner/provisioner_test.go

```diff
@@ -1860,17 +1860,6 @@ func (s *withControllerSuite) TestAPIAddresses(c *gc.C) {
 	})
 }
 
-func (s *withControllerSuite) TestStateAddresses(c *gc.C) {
```

Comment:
This breaks backward compatibility, even if unused.
I think the version needs to be bumped and the method stubbed for old versions as was done in:
https://github.com/juju/juju/pull/11706